source/includes/common/bucket-replication.rst

```diff
@@ -0,0 +1,181 @@
+        - The maximum length of time in seconds MinIO should spend verifying the health of the replicated data on the destination bucket.
+
+      * - Storage Class
+        - The class of storage to use on the destination deployment for the replicated data.
+          Valid values are either ``STANDARD`` or ``REDUCED_REDUNDANCY``.
+
+      * - Object Filters
+        - Limit which objects to replicate from the bucket by :term:`Prefix` or **tags**.
+          If you enter multiple tags, the objects must match all tag values.
+
+      * - Metadata Sync
+        - Leave selected to also replicate the object's metadata file.
+          Otherwise, move the toggle to the :guilabel:`Off` position.
+
+      * - Delete Markers
+        - Leave selected to also replicate MinIO's indication that an object has been deleted and should also be marked deleted at the ation bucket.
+          Otherwise, move the toggle to the :guilabel:`Off` position to prevent marking the object as deleted in the destination bucket.
+
+      * - Deletes
+        - Leave selected to allow replication of the deletion of versions of an object.
+          Otherwise, move the toggle to the :guilabel:`Off` position to not replicate deletion of object versions.
+
+#. Select :guilabel:`Save` to finish adding the replication rule
+
+.. end-create-bucket-replication-rule-console-desc
+
+
+.. start-create-bucket-replication-rule-cli-desc
+
+Use the :mc-cmd:`mc replicate add` command to add a new replication rule to each MinIO deployment. 
+
+.. code-block:: shell
+   :class: copyable
+
+   mc replicate add ALIAS/BUCKET \
+      --remote-bucket 'arn:minio:replication::<UUID>:DESTINATIONBUCKET' \
+      --replicate "delete,delete-marker,existing-objects"
+
+- Replace ``ALIAS`` with the :ref:`alias <alias>` of the origin MinIO deployment.  
+  The name *must* match the bucket specified when creating the remote target in the previous step.
+
+- Replace ``BUCKET`` with the name of the bucket to replicate from on the origin deployment. 
+
+- Replace the ``--remote-bucket`` value with the ARN for the destination bucket determined in the first step. 
+  Ensure you specify the ARN created on the origin deployment. 
+  You can use :mc-cmd:`mc admin bucket remote ls` to list all remote ARNs configured on the deployment.
+
+- The ``--replicate "delete,delete-marker,existing-objects"`` flag enables the following replication features:
+  
+  - :ref:`Replication of Deletes <minio-replication-behavior-delete>` 
+  - :ref:`Replication of existing Objects <minio-replication-behavior-existing-objects>`
+  
+  See :mc-cmd:`mc replicate add --replicate` for more complete documentation. 
+  Omit any field to disable replication of that component.
+
+Specify any other supported optional arguments for :mc-cmd:`mc replicate add`.
+
+.. end-create-bucket-replication-rule-cli-desc
+
+.. start-validate-bucket-replication-console-desc
+
+#. Go to the :guilabel:`Buckets` section of the MinIO Console
```

Comment:
Same thought w.r.t. letters vs numbering. 

This might also benefit from at least one screenshot

Reply:
Updated and added two screenshots